thunderbird/MozillaThunderbird.spec

```diff
@@ -560,51 +569,49 @@ rm -rf %{_tmppath}/translations.*
 %endif
 
 %post
-# update mime and desktop database
-%mime_database_post
```

Comment:
the package does not provide a mime database registrations AFAICS. So this is a resource and performance hit at installation time